.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "hardfloat"]
 	path = hardfloat
-	url = https://github.com/ucb-bar/berkeley-hardfloat.git
+	url = https://github.com/bigzz/berkeley-hardfloat.git
```

Comment:
Why doing this? You should submit PRs to hard float rather making RocketChip depending on your own fork

Reply:
Very sorry for this, I just want to commit this changes to my own branch. Thanks for your reply！